frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -220,6 +220,16 @@ if Device:isAndroid() then
         end,
     }
 
+    -- camera key events
+    common_settings.android_camera_keys = {
+        text = _("Camera key toggles touchscreen"),
```

Comment:
Maybe something like this?

Camera key toggles ignore mode

Reply:
Since I'm not native I'm fine with something that means: "(usar la) tecla de cámara (para) conmuta(r) el panel táctil". The easy translation for that is: **camera key switches/toggles touchscreen** but if you find your suggestion a better translation then is good for me :smile:

Reply:
I find "ignore mode" a bit obscure (what is ignored? history, statistics, page turns, taps, swipes, other keys?) - and it's not all user interactions that are ignored: volume keys to turn pages would still work while touchscreen is disabled.
"toggles touchscreen activation" ?

Reply:
`Touch ignore mode (camera button)`?

Reply:
Would need the word "toggle" or "disable/enable", otherwise one may think it's the checkbox that set the mode.

Reply:
+1 to toggle or disable/enable. Feel free to correct and merge when ready :smiley:

Reply:
"Camera key touchscreen toggling", if we want to have the same grammatical construct as the previous "Volume key page turning" ? (Although even for this one, my limited english skills would more easily understand a sentence like "Use/allow volume keys to turn page" - so I'm fine with this PR that uses a sentence.)

Reply:
the first one is described as an option, the second as an action. The first sentence is saying "this to use x key to do y action" while the second assumes that "this" is an option and says "use x key to do y action".

I prefeer the second option too (in both cases).